index.Rmd

```diff
@@ -304,101 +303,116 @@ ggplot(us_data, aes(x = med_income, y = med_prop_val)) +
 
 One common and easy way to format your `ggplot` visualization is to add additional layers to the plot object using the `+` symbol. 
 For example, we can use the `xlab` and `ylab` functions to add layers where we specify human readable labels for the x and y axes. 
-Again, since we are specifying words (e.g. `"Income (USD)"`) as arguments to `xlab` and `ylab`, we surround them with double quotes.
+Again, since we are specifying words (e.g. `"Mother tongue (number of Canadian residents)"`) as arguments to `xlab` and `ylab`, we surround them with double quotes.
 There are many more layers we can add to format the plot further, and we will explore these in later chapters.
 
 <center>
-```{r prop_val_vs_income_human_labs, fig.width=4.75, fig.height=4, warning=FALSE}
-ggplot(us_data, aes(x = med_income, y = med_prop_val)) +
+```{r mother_tongue_vs_most_at_home_labs, fig.width=5.75, fig.height=5, warning=FALSE}
+ggplot(can_lang, aes(x = most_at_home, y = mother_tongue)) +
     geom_point() +
-    xlab("Income (USD)") +
-    ylab("Median property value (USD)")    
+    xlab("Language spoken most at home (number of Canadian residents)") +
+    ylab("Mother tongue (number of Canadian residents)")    
 ```
 </center>
 
-From this visualization we see that for the 52 US regions in this data set, as median household income increases so does median property value. When we see 
+Most of the data points from the 214 observations in this data set are bunched up in the lower left-handside of this visualization. This is because many many more people in Canada speak the two official languages (English and French). Thus to answer our question, we will need to adjust the scale of the x and y axes so that they are on a log scale. We can again add additional layers to the plot object using the `+` symbol to do this:
```

Comment:
```suggestion
Most of the data points from the 214 observations in this data set are bunched up in the lower-left hand side of this visualization. This is because many more people in Canada speak the two official languages (English and French). Thus to answer our question, we will need to adjust the scale of the x and y axes so that they are on a log scale. We can again add additional layers to the plot object using the `+` symbol to do this:
```